lib/App/GHPT/WorkSubmitter.pm

```diff
@@ -184,6 +184,8 @@ sub run ($self) {
         die "No started stories found!\n";
     }
 
+    system("clear");
```

Comment:
Rather than using `clear` to do this, it would be better to use a module. For instance, you can [supply an option](https://metacpan.org/pod/Term::Choose#clear_screen) to `choose()` to clear the screen, which likely would accomplish what you want.

Reply:
Agreed.